rpc/Cargo.toml

```diff
@@ -19,3 +22,18 @@ anyhow = "1.0.71"
 dotenvy = "0.15.7"
 rand = "0.8.5"
 tokio = { version = "1.32.0", features = ["rt", "macros"] }
+env_logger = "*"
+log = "*"
+futures = "0.3.28"
+libp2p = { version = "*", features = [
```

Comment:
define the version of the crates

Reply:
removed unnecessary dependencies and set version for the remaining `log`